core/spec/models/spree/promotion_spec.rb

```diff
@@ -64,6 +65,10 @@
     it "should delete rules" do
       expect(Spree::PromotionRule.count).to eq(0)
     end
+
+    it "should delete codes" do
+      expect(Spree::PromotionCode.count).to eq(0)
```

Comment:
@ono if you wouldn't mind removing all three of these `it "should delete xxx"` specs I think that would be great.